docs/refman/CMakeLists.txt

```diff
@@ -19,9 +19,12 @@ if (ENABLE_REFMAN)
 
   # Allow @experimental to be used in headers to mark an API as experimental and add it to a list of
   # experimental APIs.
+  # Also allow @advanced to be used in headers to mark an API as advanced and add it to a list of
+  # advanced APIs.
   set(DOXYGEN_ALIASES
     "experimental = \\xrefitem experimental \\\"This feature is marked as experimental\\\" \\\"Experimental List\\\""
-    "experimental{1} = \\xrefitem experimental \\\"Experimental\\\" \\\"Experimental List\\\" \\1")
+    "experimental{1} = \\xrefitem experimental \\\"Experimental\\\" \\\"Experimental List\\\" \\1"
+    "advanced = \\xrefitem advanced \\\"This feature is marked as advanced and is not intended for use by apps\\\" \\\"Advanced List\\\"")
```

Comment:
Don't we also need to add the per item alias? i.e.
```
"advanced{1} = \\xrefitem advanced \\\"Advanced\\\" \\\"Advanced List\\\""
```

Reply:
That would only be if we needed a 1-arg variation.  For comparison, the @experimental PR explained:

So you use no argument if the entire API is experimental.
You use an argument if some particular subset or variation is experimental.

Line 27 is sufficient if an entire API is advanced.  We would need the advanced{1} if we thought that some subset or variation (e.g., some argument being non-null) were advanced but the rest of the API were not.  I omitted it since I think we should rule out that case.

Reply:
Thanks for the explanation, I agree we should probably keep it to the entire function. It can always be added later should the need arise.